msyncd/synchronizer.cpp

```diff
@@ -184,7 +186,7 @@ bool Synchronizer::initialize()
         else
         {
             QObject::connect(&iSyncOnChangeScheduler, SIGNAL(syncNow(QString)),
-                             this, SLOT(startSync(QString)),
+                             this, SLOT(startScheduledSync(QString)),
```

Comment:
In this case, I don't think it's a scheduled sync (ie, not a sync caused by the sync schedule, but instead it's a sync caused by an account change).  So, I think it should be startSync() instead of startScheduledSync().  Do you agree, or am I misunderstanding the semantics?